jsons/Beliefs.json

```
@@ -483,7 +483,7 @@
         "name": "World Church",
         "type": "Founder",
         "uniques": [
-            "[+1 Culture] for every [2] global followers [in foreign cities]"
+            "[+1 Culture] form every [2] global followers [in foreign cities]"
```

```suggestion
            "[+1 Culture] from every [2] global followers [in foreign cities]"
```
You were so close!
Good catches!